src/psystem.jl

```diff
@@ -0,0 +1,153 @@
+module PowerSystem
+
+using ..ExaPF: Parse
+using ..ExaPF: Network
+
+using SparseArrays
+
+"""
+    Pf
+
+This structure contains constant parameters that define the topology and
+physics of the power network.
+
+# Fields
+- `V::Array{Complex{Float64}}`: voltage
+"""
+struct Pf
```

Comment:
How about renaming `Pf` `PowerFlow`?

Reply:
Renamed it to PowerNetwork (referring to the physical object)